docs/installing-viper.rst

```diff
@@ -99,3 +115,12 @@ install and test commands
     ::
         python setup.py install
         python setup.py test
+******
+Docker
+******
+A Docker image which executes most of the installation steps is provided.
+To ensure that everything works correctly after the installtion, please run the test commands
+and try compiling a contract:
+::
```

Comment:
It'd be great if you could explain how to setup docker starting with cloning the repo (e.i. `git clone ...`).

Reply:
@DavidKnott Sure, but I am not really familiar with Docker. After a quick search: `docker build https://github.com/ethereum/viper.git -t viper:v1` (Not sure if the :v1 is unnecessary but probably easier to keep track of future images v2, ...) and then  `docker run viper:v1`?

Reply:
@chgue These are the commands you need to run:
(put the `git clone viper` here)
1. `docker build -t viper .`
2. `docker run -it viper /bin/bash`
3. `viper examples/crowdfund.v.py`

Reply:
@DavidKnott I tested the ` build https://github.com/ethereum/viper.git -t viper:v1` command  yesterday and it worked so I commited that. Do you want me to change it to your steps? 

Included the other changes you mentioned as well